src/GraphQL/Response/Response.php

```diff
@@ -4,8 +4,6 @@
 
 namespace Drupal\graphql\GraphQL\Response;
 
-use Drupal\jobiqo_graphql\Wrappers\Violation\ViolationCollection;
```

Comment:
@rthideaway in your commit you removed this line. was it intentional?

Reply:
probably, this looks like a jobiqo namespace that should not be in graphql module.